_posts/it/2020-10-16-ALTF4-9.md

```diff
@@ -0,0 +1,84 @@
+---
+author: TheEnemy42, stringweasel
+editor: Therenas, nicgarner

```

Comment:
Who was the translator for this?

Reply:
erf, I forget that the IT is not doing this. I add that